src/WebJobs.Extensions.ApiHub/Listener/ApiHubListener.cs

```diff
@@ -20,13 +21,16 @@ namespace Microsoft.Azure.WebJobs.Extensions.ApiHub
     [Singleton(Mode = SingletonMode.Listener)]
     internal class ApiHubListener : IListener
     {
+        private const int MaxFunctionExecutionRetryCount = 5;
```

Comment:
I think you should make this a configuration knob on your api hub config object. For queues/blobs this is configurable at the host level